smt/surrogate_models/krg_based.py

```diff
@@ -895,6 +895,8 @@ def grad_minus_reduced_likelihood_function(log10t):
                             stop += 1
                             if incr != 0:
                                 return
+                            if stop > max_retry:
+                                raise ValueError('%d attempt to train the model failed' %max_retry)
```

Comment:
attempts and would you address previous review minor requested change?

Reply:
Ok no problem